docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,264 @@
+---
+approvers:
+- mtaufen
+- dawnchen
+title: Reconfigure a Node's Kubelet in a Live Cluster
+---
+
+{% capture overview %}
+As of Kubernetes 1.8, the new [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281) feature is available in alpha. This allows you to change the configuration of Kubelets in a live Kubernetes cluster via first-class Kubernetes concepts. Specifically, this feature allows you to configure individual Nodes' Kubelets via ConfigMaps.
+
+**Warning:** The organization of the Kubelet's configuration type is still considered alpha, and it is likely to be restructured with breaking changes before graduating from alpha.
+
+**Warning:** All Kubelet configuration parameters may be changed dynamically, but not all parameters are safe to change dynamically. This feature is intended for system experts who have a strong understanding of how configuration changes will affect behavior. No documentation currently exists which plainly lists "safe to change" fields, but we plan to add it before this feature graduates from alpha.
```

Comment:
Some things, like the kubeletconfig API group, might graduate directly to GA in 1.9 (this would make sense, as the set of knobs in that group is representative of the already-GA flags API).

Reply:
Writing graduating to GA makes sense to me